### ISSUE TITLE
Keyboard support: Focus order / missing datetimepicker.hide event

### ISSUE THREAD

**Issue by javagotchi** [Eonasdan/tempus-dominus#2955]
### Prerequisites

- [x] I have [searched](https://github.com/Eonasdan/tempus-dominus/issues?q=is%3Aissue) for duplicate or closed issues.
- [x] I have validated my setup against the latest version on [StackBlitz](https://stackblitz.com/edit/tempus-dominus-v6-simple-setup).
- [x] I have read the [contributing guidelines](https://github.com/Eonasdan/tempus-dominus/blob/master/.github/CONTRIBUTING.md).
- [x] I understand that only sponsored issues will be worked on and that if I wish to sponsor an issue, I will contact the owner FIRST.

### Describe the issue

Hello there!
Working on the accessibility of a website and was glad to see that v.6.10.2 has now keyboard support!

Unfortunately the datepicker-element loses the focus after been closed and the focus continues on the end of the document where the widget is placed. So other important inputs and links are jumped over.

I tried to fix this issue by setting the focus manually but unfortunately the "hide" event is not fired either.

this works:

$('#datetimepicker1').on('change.datetimepicker',(e)=>{console.log(e); });
$('#datetimepicker1').on('change',(e)=>{console.log(e); });

this not:
$('#datetimepicker1').on('datetimepicker.hide',(e)=>{console.log(e); });
$('#datetimepicker1').on('hide.datetimepicker',(e)=>{console.log(e); });
$('#datetimepicker1').on('hide',(e)=>{console.log(e); });

 We would be lucky to continue using tempus-dominus because it looks much better than native browser elements.
Thanks for your help!

### StackBlitz fork

const datetimepicker1 = new tempusDominus.TempusDominus(
document.getElementById('datetimepicker1'),
  {
	allowInputToggle:true,
    display: {
		sideBySide: true,
		icons: {
		type: 'icons',
		time: 'fas fa-clock',
		date: 'fas fa-calendar-alt',
		up: 'fas fa-arrow-up',
		down: 'fas fa-arrow-down',
		previous: 'fas fa-chevron-left',
		next: 'fas fa-chevron-right',
		today: 'fas fa-calendar-check',
		clear: 'fas fa-trash',
		close: 'fas fa-check'
		},
		buttons: {
        today: false,
        clear: false,
        close: true
      },
	},
		stepping: 15,	
		restrictions: {
			minDate: currenttime ,
		},
  }
);

$('#datetimepicker1').on('hide.datetimepicker',(e)=>{console.log(e); });

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of are you using? You can find this information from the sample StackBlitz.

v.6.10.2

### What your browser's locale? You can find this information from the sample StackBlitz.

de-DE

**Comment by javagotchi** [comment]
additional: in the sideBySide layout I wasn't able to get to the time-menu with keyboard

**Comment by Eonasdan** [comment]
Hi. The events you are looking for are [here](https://getdatepicker.com/6/namespace/events.html#hide)

When the picker closes, it will return focus to the toggle button. I've updated the code to place focus back in the input field if the toggle button is not present. That will go out soon. You can try it [out here](https://stackblitz.com/edit/tempus-dominus-v6-simple-setup-ky4mljz).

I spent a few minutes trying to see if I could improve the keyboard nav in side by side mode, but I didn't have time to make it work completely. The keyboard navigation was sponsored and this wasn't a scenario that was brought up.

If you're interested in sponsoring additional work for the side by side, please let me know.
